### PR TITLE
Discover 2

### PR DESCRIPTION
**Pull request by shaybrow**

### CODE REVIEW

File: src/main/java/com/islandcollaborative/creativeexchange/repositories/AppUserRepository.java [hunk 1]
```diff
@@ -4,9 +4,14 @@
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
+import javax.swing.text.StyledEditorKit;
```
Comment (wordhou) [inline src/main/java/com/islandcollaborative/creativeexchange/repositories/AppUserRepository.java:7]: Not sure this import belongs?

File: src/main/resources/templates/discover.html [hunk 2]
```diff
@@ -4,10 +4,11 @@
 <body>
 <header th:replace="fragments/layout :: header"> </header>
 
-<section th:each=" user: ${users)">
+<section th:each=" user: ${users}">
```
Comment (wordhou) [inline src/main/resources/templates/discover.html:7]: This is a good start, should we fragmentize this into a `fragments/user.html` file?

Comment (wordhou) [reply src/main/resources/templates/discover.html:7]: Maybe in a future PR

**Review by wordhou — APPROVED** [review]

File: src/main/java/com/islandcollaborative/creativeexchange/repositories/AppUserRepository.java (hunk 1 above)
Comment (shaybrow) [reply src/main/java/com/islandcollaborative/creativeexchange/repositories/AppUserRepository.java:7]: agreed. let's delete it

**Pull request merged**